src/components/Pages/RequestPage/RequestPage.tsx

```diff
@@ -251,12 +262,20 @@ export const RequestPage = () => {
       await authServerClient.current.sendSuccessfulOutcome(requestId, await signer.getAddress(), signature)
       console.log('Approve sign in verification - Outcome sent')
 
+      if (signTimeoutRef.current) {
```

Comment:
If I'm not wrong, we should set the timeout before the line:
`const signature = await signer.signMessage(requestRef.current?.params?.[0])`
and after it we should remove the timeout.

The timeout should drop the whole operation though, so we should store a variable before the sign out timeout to be checked after the signMessage line, like:
```
let hasTimeouted = false
signTimeoutRef.current = setTimeout(() => {
    hasTimeouted = true
    setHasTimedOut(true)
    setIsLoading(false)
})
const signature = await signer.signMessage(requestRef.current?.params?.[0])
if (hasTimeouted) {
  throw TimedOutError()
}
```
This is because if the user eventually sees that there's an operation to be signed, but the process has been timeout, we should not do anything about it.